pax_experiments/if3c_htm/if3c_experiments_150122.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python
+
+# -*- coding: utf-8 -*-
+
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2013, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""
+Created on Thu Jan 22 16:14:22 2015
+
+Experiments creating HTM with multi-compartment IF neurons. Note that this 
+script requires brian2 and pylab. 
+
+@author: pfrady
+"""
+
+
+
+from pylab import *
+from brian2 import *
+
+#%% 3 Compartment Excitatory neuron params
```

Comment:
A comment describing at a high level what this script is doing would be helpful.